### ISSUE TITLE
Keycloak new GUI support

### ISSUE THREAD

**Issue by xsuchy09** [grnet/eidas-keycloak-extension#6]
First of all thanks for all the job you have done.

Is there any plan to support new GUI of Keycloak? There is no support from version 18.0.2 (19+) at least for initial setup.

**Comment by d-michail** [comment]
Updated the codebase for keycloak 24.0.4. Now the extra attributes appear in the new admin UI, and you do not have to set them up using some low-level technique (json import or directly from the database). 

Still two (minor) caveats related to the new admin UI
 - When you first setup the provider it displays a clientId and a clientSecret which are not used anywhere. Just include some fake values and ignore. This is related to https://github.com/keycloak/keycloak/pull/24266 . 
 - The new keycloak can help users by reading the configuration from a url (metadata descriptor). This is hardcoded in the admin UI, so it does appear.

**Comment by cesarcoruna** [comment]
Those attributes only appear in the momment you create the IDP. When you are editing an exinsting one they don't appear.
![kc24-options](https://github.com/grnet/eidas-keycloak-extension/assets/144889482/0afc47fc-1f2a-44ad-b184-bbb2e64c65b1)

**Comment by cesarcoruna** [comment]
Yes, you are right. That position is very different that it is in documentation and, because of that, i didn't see it. Thanks
